playbooks/roles/forum/tasks/deploy.yml

```
@@ -84,6 +84,14 @@
     - migrate
     - migrate:db
 
+- name: MongoDB Indexing for cs_comments_service
```

```suggestion
- name: rebuild MongoDB indexes
```

Since this is the `forum` role, other data is redundant.